export/export_plugin.cpp

```diff
@@ -0,0 +1,64 @@
+#include "export_plugin.h"
+
+//EditorExportPlatformWiiU::EditorExportPlatformWiiU(){
+
+//};
+
+void EditorExportPlatformWiiU::get_preset_features(const Ref<EditorExportPreset> &p_preset, List<String> *r_features) const {
+
+};
+
+void EditorExportPlatformWiiU::get_export_options(List<ExportOption> *r_options) const {
+
+};
+
+String EditorExportPlatformWiiU::get_name() const {
+    return "Wii U";
+}
+
+String EditorExportPlatformWiiU::get_os_name() const { 
+	return "Cafe OS"; 
+}
+
+Ref<Texture2D> EditorExportPlatformWiiU::get_logo() const {
+    return logo;
+}
+
+Ref<Texture2D> EditorExportPlatformWiiU::get_run_icon() const {
+    return run_icon;
+}
+
+bool EditorExportPlatformWiiU::has_valid_export_configuration(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates, bool p_debug) const {
+    return false;
+}
+
+bool EditorExportPlatformWiiU::has_valid_project_configuration(const Ref<EditorExportPreset> &p_preset, String &r_error) const {
+    return false;
+}
+
+//TODO: Fix this; this is literally copy-pasted from IOS with a line change, lol
+List<String> EditorExportPlatformWiiU::get_binary_extensions(const Ref<EditorExportPreset> &p_preset) const {
+	List<String> list;
+	if (p_preset.is_valid()) {
+		bool project_only = p_preset->get("application/export_project_only");
+		if (project_only) {
+			list.push_back("xcodeproj");
+		} else {
+			list.push_back("rpx");
+		}
+	}
+	return list;
+}
+
+Error EditorExportPlatformWiiU::export_project(const Ref<EditorExportPreset> &p_preset, bool p_debug, const String &p_path, int p_flags){
+    return ERR_CANT_CREATE;
+}
+
+void EditorExportPlatformWiiU::get_platform_features(List<String> *r_features) const {
+		r_features->push_back("game_console");
+		r_features->push_back("homebrew");
```

Comment:
Are these completely random? What is the point of these?

Reply:
I was mimicking the IOS platform implementation for this, where they listed the features "mobile" and "ios", so I was basically just mimicking that. It could be changed, idk what these fields are even used for besides their listing in the editor export

Reply:
If it is completely random, I say we should add useful stuff, like `wii_pointer` so we can query from game code.

Reply:
Sounds good. Any thoughts for what else to list there?

Also, I'm not entirely sure how I would wire up `wii_pointer` between the compiler and the code yet, if you're referring to what I have in `detect.py`. Those options are more or less just placeholders; I haven't looked into how to actually use those options to manipulate what code compiles yet.

Reply:
No I didn't even look at detect.py. I have no idea what you're thinking. I just think it would be a good idea to be able to check if the platform supports things like the Wii pointer from code.

Reply:
Ah, alright. I'll add that in, then.